lib/components/formik/Form.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Formik, Form as FormikForm } from "formik";
+import PropTypes from "prop-types";
+
+const Form = ({ children, formikProps, formProps }) => (
+  <Formik {...formikProps}>
+    {() => (
+      <FormikForm noValidate {...formProps}>
+        {children}
+      </FormikForm>
+    )}
+  </Formik>
+);
```

Comment:
I think we would need to pass down the `Formik` child props for handling cases like below, WDYT?

```js
     <Formik
       initialValues={{ name: 'jared' }}
       onSubmit={(values, actions) => {
         setTimeout(() => {
           alert(JSON.stringify(values, null, 2));
           actions.setSubmitting(false);
         }, 1000);
       }}
     >
       {props => (
         <form onSubmit={props.handleSubmit}>
           <input
             type="text"
             onChange={props.handleChange}
             onBlur={props.handleBlur}
             value={props.values.name}
             name="name"
           />
           {props.errors.name && <div id="feedback">{props.errors.name}</div>}
           <button type="submit">Submit</button>
         </form>
       )}
     </Formik>
```